ember-cli-build.js

```diff
@@ -3,7 +3,11 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  let app = new EmberAddon(defaults, {});
+  let app = new EmberAddon(defaults, {
+    'ember-cli-babel': {
+      throwUnlessParallelizable: true,
```

Comment:
This helps ensure we (when developing locally in ember-data) don't accidentally regress the ability to do parallel builds...